R/correlationbayesian.R

```diff
@@ -86,20 +86,27 @@ CorrelationBayesian <- function(jaspResults, dataset=NULL, options, ...) {
         v1 <- NA
         v2 <- NA
       } else {
-        v1 <- dataset[[.v(var1)]]
-        v2 <- dataset[[.v(var2)]]
+        v1 <- dataset[[jaspBase::encodeColNames(var1)]]
+        v2 <- dataset[[jaspBase::encodeColNames(var2)]]
```

Comment:
are you sure you need the encoding at all? assuming `var1` and `var2` come from an options list, they should be already encoded by JASP.